src/client/common/application/customEditorService.ts

```diff
@@ -44,35 +44,43 @@ export class CustomEditorService implements ICustomEditorService {
-        const editorAssociations = settings.get('editorAssociations') as {
-            viewType: string;
-            filenamePattern: string;
-        }[];
+    public async activate() {
+        let updateType: 'added' | 'removed' | undefined;
+        try {
+            // This code is temporary.
+            const settings = this.workspace.getConfiguration('workbench', undefined);
+            const editorAssociations = settings.get('editorAssociations') as {
+                viewType: string;
+                filenamePattern: string;
+            }[];
 
-        // Update the settings.
-        if (
-            this.useCustomEditorApi &&
-            (editorAssociations.length === 0 || !editorAssociations.find((item) => item.viewType === ViewType))
-        ) {
-            editorAssociations.push({
-                viewType: ViewType,
-                filenamePattern: '*.ipynb'
-            });
-            await settings.update('editorAssociations', editorAssociations, vscode.ConfigurationTarget.Global);
-        }
+            // Update the settings.
+            if (
+                this.useCustomEditorApi &&
+                (editorAssociations.length === 0 || !editorAssociations.find((item) => item.viewType === ViewType))
+            ) {
+                editorAssociations.push({
+                    viewType: ViewType,
+                    filenamePattern: '*.ipynb'
+                });
+                updateType = 'added';
+                await settings.update('editorAssociations', editorAssociations, vscode.ConfigurationTarget.Global);
+                sendTelemetryEvent(Telemetry.UpdateCustomEditorAssociation, undefined, { type: 'added' });
+            }
 
-        // Revert the settings.
-        if (
-            !this.useCustomEditorApi &&
-            Array.isArray(editorAssociations) &&
-            editorAssociations.find((item) => item.viewType === ViewType)
-        ) {
-            const updatedSettings = editorAssociations.filter((item) => item.viewType !== ViewType);
-            await settings.update('editorAssociations', updatedSettings, vscode.ConfigurationTarget.Global);
+            // Revert the settings.
+            if (
+                !this.useCustomEditorApi &&
+                Array.isArray(editorAssociations) &&
+                editorAssociations.find((item) => item.viewType === ViewType)
+            ) {
+                const updatedSettings = editorAssociations.filter((item) => item.viewType !== ViewType);
+                updateType = 'removed';
+                await settings.update('editorAssociations', updatedSettings, vscode.ConfigurationTarget.Global);
+                sendTelemetryEvent(Telemetry.UpdateCustomEditorAssociation, undefined, { type: 'removed' });
+            }
+        } catch (ex) {
+            sendTelemetryEvent(Telemetry.UpdateCustomEditorAssociation, undefined, { type: updateType! }, ex, true);
```

Comment:
At least 2 users ran into this issue and could not open notebooks. Possible there are more users.